compiler-plugin/src/main/java/io/ballerina/stdlib/log/compiler/LogCompilerPlugin.java

```diff
@@ -16,18 +16,27 @@
  * under the License.
  */
 
-package io.ballerina.stdlib.log.plugin;
+package io.ballerina.stdlib.log.compiler;
 
 import io.ballerina.projects.plugins.CompilerPlugin;
 import io.ballerina.projects.plugins.CompilerPluginContext;
+import io.ballerina.scan.ScannerContext;
+import io.ballerina.stdlib.log.compiler.staticcodeanalyzer.StaticCodeAnalyzer;
 
 /**
  * log module Compiler plugin.
  */
 public class LogCompilerPlugin extends CompilerPlugin {
 
+    private static final String SCANNER_CONTEXT = "ScannerContext";
+
     @Override
-    public void init(CompilerPluginContext compilerPluginContext) {
-//        compilerPluginContext.addCodeModifier(new LogCodeModifier());
+    public void init(CompilerPluginContext context) {
+
+        //context.addCodeModifier(new LogCodeModifier());
```

Comment:
```suggestion
```

Reply:
Fixed in [3edb530](https://github.com/ballerina-platform/module-ballerina-log/pull/1175/commits/3edb5309e84fdca1986162244a181483a3bf3b2e)